### DISCUSSION TITLE
Customise the name of the function when you add in the .md file

### DISCUSSION THREAD

**Discussion by JeyDi** [mkdocstrings/mkdocstrings#476] [Q&A] [answered]
Hi! There is a way to customise easily the name of the code reference (function or class) to exclude the root path in the table of content and in the page?

For example:

::: mypackage.src.db.query.publication.count.month.data

to display only: *Data* (the name of the function)  in the page and especially in the table of content?

I tried this configurations, but it didn't work:

::: mypackage.src.db.query.publication.count.month.data
    options:
        show_root_heading: true
        show_root_full_path: false
        show_source: false
        heading_level: 4

**Comment by pawamoy** [comment, marked as answer]
Hello!

I believe what you want is `show_object_full_path: false`. It's documented here: https://mkdocstrings.github.io/python/usage/#globallocal-options
Let me know if that works for you.

**Reply by pawamoy** [reply]
Also look at the example in this comment: https://github.com/mkdocstrings/python/issues/79#issuecomment-1645768652
